providers/common/include/internal/providercommon.h

```diff
@@ -16,50 +16,72 @@ const char *ossl_prov_util_nid_to_name(int nid);
 
 typedef struct {
     /*
-     * References to the underlying cipher implementation.  cipher caches
-     * the cipher, always.  alloc_cipher only holds a reference to an explicitly
-     * fetched cipher.
-     * cipher is cleared after an init call.
+     * References to the underlying cipher implementation.  _cipher_ caches
+     * the cipher, always.  _alloc_cipher_ only holds a reference to an
+     * explicitly fetched cipher.
```

Comment:
In other places in our code, we have | around variable references in comments...

Reply:
Easy change, will do.

Reply:
& done :)

Reply:
I did this when I made a new file for the new functions.